package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "egeria-ui",
+  "version": "0.0.1",
+  "description": "Egeria UI application",
+  "author": "Egeria contributors",
+  "license": "Apache-2.0",
+  "repository": "odpi/egeria-ui",
+  "vaadin": {
+    "disableUsageStatistics": true
+  },
+  "scripts": {
+    "serve": "hs -c0 ./build/dev/ -P http://localhost:9000",
```

Comment:
I'm not a node expert -- will these work ok on windows? Is node smart enough?

Reply:
> I'm not a node expert -- will these work ok on windows? Is node smart enough?

it's not node, it's `http-serve` npm package

Reply:
I think for now as long as there are instructions on how a developer can test that covers linux, mac, windows, that is good enough

By node I was referring to the unix specific command - we need to make sure we are not dependent on it so that we can still work on windows (including developers)